src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -247,6 +251,38 @@ protected override void NodeModified(NodeModel node)
             }
         }
 
+        /// <summary>
+        /// Called when a node is added to the workspace and event handlers are to be added
+        /// </summary>
+        /// <param name="node">The node itself</param>
+        protected virtual void RegisterNode(NodeModel node)
```

Comment:
How is this not `override`? This method signature is the same as the one in `WorkspaceModel`.